active: false
main_timer_rate: 100
max_distance: 120
height: 1.5
randomize_distance: false
